alacritty_terminal/src/window.rs

```diff
@@ -387,6 +365,11 @@ impl Window {
         }
     }
 
+    #[cfg(not(any(target_os = "macos", target_os = "windows")))]
+    pub fn x11_window(&self) -> bool {
+        self.event_loop.is_x11()
+    }
```

Comment:
I think the method name could probably be improved, since `window.x11_window()` is a bit of unnecessary duplication. I'd probably just go with `window.is_x11` to stay consistent with the glutin API's name.

Alternatively since Display::new has access to the event_loop too, before the window is created, we could store the x11 state in a variable before creating the window and query that later, but I think it's nicer to expose this method on the window. I'm also not sure how this is handled in the evlp2 branch, but it's possible that things need to be changed there anyways, so I wouldn't worry *too* much.